nodeInterface/NodeInterface.go

```diff
@@ -476,7 +476,7 @@ func (n NodeInterface) GasEstimateComponents(
 	if err != nil {
 		return 0, 0, nil, nil, err
 	}
-	l1BaseFeeEstimate, err := pricing.L1BaseFeeEstimate()
+	l1BaseFeeEstimate, err := pricing.PricePerUnit()
```

Comment:
I'd prefer to stick with `PricePerUnit` because it's not an estimate of the L1 gas price--for an AnyTrust chain this price should be much less than the L1 gas price.